packages/bot-skeleton/src/scratch/xml/main.xml

```diff
@@ -1,4 +1,4 @@
-﻿<xml xmlns="http://www.w3.org/1999/xhtml" collection="false">
+﻿<xml xmlns="http://www.w3.org/1999/xhtml" collection="false" is_dbot="true">
```

Comment:
@farabi-deriv can we use this way?
```suggestion
﻿<xml xmlns="http://www.w3.org/1999/xhtml" collection="false" is_dbot>
```

Reply:
@maryia-matskevich-deriv i found that all the quick strategies are using `is_dbot="true"`, also when i realign the default xml and download it, it contains `is_dbot="true"`. Thats why i have added this as default